README.rst

```diff
@@ -1,84 +1,121 @@
-    # and SIGNALFX_TRACING_URL if set
-    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
+1. Open your AWS console. 
+2. In the landing page, under **Compute**, click **Lambda**.
+3. Click **Create function** to create a layer with SignalFx's capabilities.
+4. Click **Browse serverless app repository**.
+5. Click **Public applications**.
+6. In the search field, enter and select **signalfx-lambda-python-wrapper**.
+7. Review the template, permissions, licenses, and then click **Deploy**.
+    * A copy of the layer will now be deployed into your account.
+8. Return to the previous screen to add a layer to the function, select from list of runtime compatible layers, and then select the name of the copy.  
 
-    # optional metrics and tracing configuration
 
-    SIGNALFX_METRICS_URL=ingest endpoint [ default: https://pops.signalfx.com ]
-    SIGNALFX_SEND_TIMEOUT=timeout in seconds for sending datapoint [ default: 0.3 ]
+Step 2: Locate and set the ingest endpoint
+-------------------------------------------
+By default, this function wrapper will send data to the us0 realm. If you are not in us0 realm and you want to use the ingest endpoint directly, then you muset set your realm. 
 
-    SIGNALFX_TRACING_URL=tracing endpoint [ default: https://ingest.signalfx.com/v1/trace ]
+To set your realm, use a subdomain, such as ingest.us1.signalfx.com or ingest.eu0.signalfx.com.
 
-``SIGNALFX_ENDPOINT_URL`` can be used to configure a common endpoint for metrics and
-traces, as is the case when forwarding with the Smart Gateway. The path ``/v1/trace``
-will automatically be added to the endpoint for traces.
+To locate your realm:
 
-If either ``SIGNALFX_TRACING_URL`` or ``SIGNALFX_METRICS_URL`` are set, they will take
-precendence over ``SIGNALFX_ENDPOINT_URL`` for their respective components.
+1. Open SignalFx and in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
 
-For example, if only ``SIGNALFX_ENDPOINT_URL`` is set:
+
+Step 3: Set environment variables
+----------------------------------
+
+Please note for environment variables, SignalFx defaults to the us0 realm. As a result, if you are not in the us0 realm, you may need to set your environment variables. 
+
+1. Set SIGNALFX_ACCESS_TOKEN with your correct access token. Review the following example. 
 
 .. code:: bash
 
-    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
+    SIGNALFX_ACCESS_TOKEN=access token
 
-Both metrics and traces will be sent to the gateway address.
 
-If ``SIGNALFX_ENDPOINT_URL`` and ``SIGNALFX_METRICS_URL`` are set:
+2. If you use POPS, Smart Gatway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (SignalFx defaults to the us0 realm. As a result, if you are in a different realm, you must explictly set the realm.) You can update one of the varibles below. Review the following examples.  
```

Comment:
```suggestion
2. If you use POPS, Smart Gateway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (SignalFx defaults to the us0 realm. As a result, if you are in a different realm, you must explicitly set the realm.) You can update one of the variables below. Review the following examples.  
```

Reply:
hyperlinks for POPS and Smart Gateway would be nice - 

https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.signalfx.point.of.presence.service.(pops).html

https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html

Reply:
Added the links, as well as corrected the misspelled 'gateway'